src/bin/tikv-server.rs

```diff
@@ -385,17 +388,23 @@ fn get_rocksdb_default_cf_option(config: &toml::Value) -> RocksdbOptions {
     get_rocksdb_cf_option(config,
                           "defaultcf",
                           1024 * 1024 * 1024,
-                          true /* bloom filter */)
+                          true, // bloom filter
+                          true /* whole key filtering */)
 }
 
 fn get_rocksdb_write_cf_option(config: &toml::Value) -> RocksdbOptions {
-    // Don't need set bloom filter for write cf, because we use seek to get the correct
-    // version base on provided timestamp.
-    get_rocksdb_cf_option(config, "writecf", 256 * 1024 * 1024, false)
+    let mut opt = get_rocksdb_cf_option(config, "writecf", 256 * 1024 * 1024, true, false);
+    // prefix extractor(trim the timestamp at tail) for write cf.
+    opt.set_prefix_extractor("FixedSuffixSliceTransform",
+                              Box::new(rocksdb_util::FixedSuffixSliceTransform::new(8)))
+        .unwrap();
+    // create prefix bloom for memtable.
+    opt.set_memtable_prefix_bloom_size_ratio(0.1 as f64);
+    opt
 }
 
 fn get_rocksdb_raftlog_cf_option(config: &toml::Value) -> RocksdbOptions {
-    get_rocksdb_cf_option(config, "raftcf", 256 * 1024 * 1024, false)
+    get_rocksdb_cf_option(config, "raftcf", 256 * 1024 * 1024, false, false)
```

Comment:
There is no bloom filter in `raft cf`, so the value of whole_key_filtering is meaningless.